clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -382,11 +382,15 @@ private String responseDataToLogString(FetchResponse<?> response) {
      */
     public boolean handleResponse(FetchResponse<?> response) {
         if (response.error() != Errors.NONE) {
-            log.info("Node {} was unable to process the fetch request with {}: {}.",
-                node, nextMetadata, response.error());
             if (response.error() == Errors.FETCH_SESSION_ID_NOT_FOUND) {
+                log.debug("Node {} was unable to process the fetch request with {} due to a cache miss. " +
```

Comment:
I think in previous discussions, we also wanted to add some metrics for this as an indication of not enough fetch sessions. Feel free to talk to @stephans for more details.

Reply:
[https://github.com/linkedin/kafka/pull/383](PR #383) adds a server-side metric for this.

Still working out how to pipe metrics into/out of this class on the client side.

Reply:
Server-side metric got submitted. Piping in the client-side metric for this is tricky.
Perhaps this PR should go through and we can revisit adding a server-side metric?
We need to stop bugging clients with this message, since it's not actionable to them.